search/search_repository.go

```diff
@@ -481,7 +481,7 @@ func generateSQLSearchInfo(keywords searchKeyword) (sqlParameter string) {
 // extracted this function from List() in order to close the rows object with "defer" for more readability
 // workaround for https://github.com/lib/pq/issues/81
 func (r *GormSearchRepository) search(ctx context.Context, sqlSearchQueryParameter string, workItemTypes []uuid.UUID, start *int, limit *int, spaceID *string) ([]workitem.WorkItemStorage, uint64, error) {
-	log.Info(ctx, nil, "Searching work items...")
+	defer r.db.LogMode(false)
```

Comment:
?

Reply:
never mind, I forgot to remove that line ;) (there was a `r.db.LogMode(true)` on top of this one, which I used to check some requests)